src/java.base/share/classes/module-info.java

```diff
@@ -273,12 +273,17 @@
         java.naming,
         java.rmi,
         java.net.http,
+        java.scripting,
         jdk.charsets,
+        jdk.hotspot.agent,
         jdk.incubator.vector,
         jdk.internal.vm.ci,
         jdk.httpserver,
+        jdk.jdi,
+        jdk.jfr,
         jdk.jlink,
         jdk.jpackage,
+        jdk.jshell,
         jdk.net;
```

Comment:
At some point we will need to re-visit all these qualified exports so that java.base exports as few of these internal packages as possible.

Reply:
Thanks, Alan and Roger. Limited the modification within the `java.base` module.